libs/core/mappers/anime.mapper.ts

```diff
@@ -0,0 +1,42 @@
+import { AnimeDto, AnimeStatusDto, AnimeDtoTypes } from '../dtos/anime.dto';
+import { Anime, AnimeTypes } from '../models/anime';
+import { AnimeStatus } from '../models/anime-status';
+
+/** Anime Mapper. */
+export namespace AnimeMapper {
+	const ANIME_TYPE_FROM_DTO: Readonly<Record<AnimeDtoTypes, AnimeTypes>> = {
+		[AnimeDtoTypes.MUSIC]: AnimeTypes.MUSIC,
+		[AnimeDtoTypes.OVA]: AnimeTypes.OVA,
+		[AnimeDtoTypes.ONA]: AnimeTypes.ONA,
+		[AnimeDtoTypes.SPECIAL]: AnimeTypes.SPECIAL,
+		[AnimeDtoTypes.TV]: AnimeTypes.TV,
+		[AnimeDtoTypes.UNKNOWN]: AnimeTypes.UNKNOWN,
+		[AnimeDtoTypes.MOVIE]: AnimeTypes.MOVIE,
+	};
+
+	const ANIME_STATUS_FROM_DTO: Readonly<Record<AnimeStatusDto, AnimeStatus>> = {
+		[AnimeStatusDto.FINISHED]: AnimeStatus.FINISHED,
+		[AnimeStatusDto.NOT_YET_AIRED]: AnimeStatus.NOT_YET_AIRED,
+		[AnimeStatusDto.AIRING]: AnimeStatus.AIRING,
+	};
+
+	/**
+	 * Converts anime DTO to anime model.
+	 * @param dto Anime GTO.
+	 */
+	export function fromAnimeDto(dto: AnimeDto): Anime {
```

Comment:
We can unify naming for each mapper functions since they will be the same
```suggestion
	export function fromDto(dto: AnimeDto): Anime {
```